standard/expressions.md

```diff
@@ -3688,6 +3815,55 @@ At run-time, the expression `await t` is evaluated as follows:
 
 An awaiter’s implementation of the interface methods `INotifyCompletion.OnCompleted` and `ICriticalNotifyCompletion.UnsafeOnCompleted` should cause the delegate `r` to be invoked at most once. Otherwise, the behavior of the enclosing async function is undefined.
 
+## §range-operator Range operator
+
+This operator provides a succinct syntax for specifying a (possibly empty) element range suitable for use in denoting a slice of an indexable sequence (§indexable-sequence).
+
+```ANTLR
+range_expression
+    : unary_expression
+    | range_expression? '..' range_expression?
+    ;
```

Comment:
This is triggering the left recursive issue in the grammar. I defer to @Nigel-Ecma for a suggestion on an alternative expression.

Reply:
Also, I don't think this grammar is correct. Would it permit something like `P[0..5..8..^2]`?

Reply:
Agree, the grammar is wrong, see later comment.